src/components/vars.sass

```diff
@@ -321,8 +321,8 @@ $tab-section-border-width: 2px
 $workspace-content-margin: 4px
 $problem-header-height: 38px
 
-$nav-expanded-peek: 14px
-$half-nav-expanded-peek: $nav-expanded-peek / 2
+$nav-expanded-peek: 12px
+$half-nav-expanded-peek: $nav-expanded-peek / 3
```

Comment:
Is this still used?  I thought this was obsolete. If used, `$half-nav-expanded-peek: $nav-expanded-peek / 3` seems odd to me since half-nav-expanded-peak is now 1/3 of the nav-expanded-peak.